src/NIDAQ.jl

```diff
@@ -41,12 +41,12 @@ export Bool32
 
 try
   global ver
-  major = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMajorVersion,NIDAQmx),Int32,(Ptr{Uint32},),major)
-  minor = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMinorVersion,NIDAQmx),Int32,(Ptr{Uint32},),minor)
-  update = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQUpdateVersion,NIDAQmx),Int32,(Ptr{Uint32},),update)
+  major = UInt32[0]
```

Comment:
the 0.4 syntax for this is now ```foo = Ref{UInt32}(0)```, then ```foo[]``` to access the returned value.  see http://docs.julialang.org/en/latest/manual/calling-c-and-fortran-code/#passing-pointers-for-modifying-inputs.  also, the corresponding ```ccall``` needs to specify Ref{UInt32} as the return type.  given that the latter requires extensive changes to src/functions_V*, i'd (again) suggest ditching support for those old versions of NIDAQmx and use Clang.jl to add support for 15.1.